test/extensions/filters/network/kafka/serialization_test.cc

```diff
@@ -151,7 +154,76 @@ TEST(VarUInt32Deserializer, ShouldThrowIfNoEndWith5Bytes) {
 
   // when
   // then
-  EXPECT_THROW(testee.feed(data), EnvoyException);
+  EXPECT_THROW_WITH_REGEX(testee.feed(data), EnvoyException, "is too long");
+}
+
+// Missing tests (chunks).
```

Comment:
is this a todo?

Reply:
Yes - sorry about that! More tests have been added.